lib/lita/state.rb

```diff
@@ -0,0 +1,43 @@
+require_relative "common"
+
+module Lita
+  class State
+    # A handler class that gives in-memory, non-persited, object storage
+    # @since 5.0.0
```

Comment:
You can move both the class description and since tag above the class declaration and then it will apply to the whole class, so you can remove the since tags from the individual methods.